docs/generate_api_reference.py

```diff
@@ -133,4 +133,14 @@ def write_package_doc(package_name):
 
 
 if __name__ == "__main__":
-    write_package_doc("gnomad_hail")
+    packages = ["resources", "utils"]
```

Comment:
I'm not 100% sure I understand this so just wanted to double check: resources and utils will still be under the same package, right? Or is this separating them into two?

Reply:
They will be distributed together. pip installing `gnomad` will get you both `gnomad.resources` and `gnomad.utils`.

Reply:
However, since now the top level `gnomad` is a namespace instead of a package, it has to be handled differently here.

Reply:
all right, sounds good! And the docs will still be available in the same format together?

Reply:
Yes

Reply:
sweet!